ceph.spec.in

```diff
@@ -20,18 +20,20 @@
 %endif
 %bcond_with make_check
 
-%ifnarch ppc64 ppc64le s390 s390x
+%ifarch aarch64 x86_64
 %bcond_without tcmalloc
-%if ! 0%{?is_opensuse}
-%bcond_without xio
-%endif
 %bcond_without valgrind_devel
 %else
 %bcond_with tcmalloc
-%bcond_with xio
 %bcond_with valgrind_devel
 %endif
 
+%if 0%{?is_opensuse}
+%bcond_with xio
+%else
+%bcond_without xio
```

Comment:
why don't you enclose this bcond under a `ifarch aarch64 x86_64`, like this:

```
%ifarch aarch64 x86_64
%bcond_without xio
%else
%bcond_with xio
%endif
```

This would allow you to avoid testing the build architecture before testing the `with xio` condition if the cases below.